README.md

```diff
@@ -195,6 +196,14 @@ $ rails server -b 0.0.0.0
 
 <hr>
 
+### Windows Installation
+We recommend you either work in a virtual environment, or on a dual booted system to avoid dependencies issues and also Unix system works smoother with Ruby and Rails. This will not only benefit you now for mapknitter, but in future working on other Ruby projects
+on as they will get into development further, a linux or mac based OS would be better. 
```

Comment:
Hi @Diegoafv, thanks for working on this...could you please remove the new line after the `Ruby projects` and also tweak it a bit to read `... but also in future when working on other Ruby projects, a Linux or Mac OS would be smoother to use...`
Thanks